drizzlepac/hapsequencer.py

```diff
@@ -404,8 +404,9 @@ def run_hap_processing(input_filename, diagnostic_mode=False, use_defaults_confi
         log.info("Return condition {}".format(return_value))
         logging.shutdown()
         # Append total trailer file (from astrodrizzle) to all total log files
-        for tot_obj in total_list:
-            proc_utils.append_trl_file(tot_obj.trl_filename, logname, clean=False)
+        if product_list:
```

Comment:
The code being turned on/off here only depend on the total_list, so it is confusing to trigger on the product_list instead.

Reply:
Fixed.